examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
```

Comment:
@DoctorMcKay I can disable the rule, if you want.
Also what do you think of arrow functions? By default, ESLint's airbnb config enforces any anonymous function to be declared as arrow functions (not `function() {}`, but `() => {}`. I've disabled the rule because I thought it was too much strict, but I'm curious of your opinion.